test/Npgsql.Tests/CopyTests.cs

```diff
@@ -602,6 +603,7 @@ public async Task Read_null_as_non_nullable_throws()
         }
 
         [Test, IssueLink("https://github.com/npgsql/npgsql/issues/1440")]
+        [NonParallelizable] // Constraint uq has to have unique name
```

Comment:
We can generate a unique constraint name, just like we generate unique table/type names etc.